packages/ckeditor5-editor-balloon/src/ballooneditoruiview.ts

```diff
@@ -63,6 +68,6 @@ export default class BalloonEditorUIView extends EditorUIView {
 		super.render();
 
 		this.registerChild( this.editable );
-		this.registerChild( this.menuBarView! );
+		this.registerChild( this.menuBarView );
```

Comment:
I cannot comment on those files, but please make the same change in `ballooneditoruiview.ts` and `multirooteditoruiview.ts`.